### PR TITLE
Add installation instructions

### LINKED ISSUE #3: add instalation instructions

### PR DESCRIPTION
**Pull request by luchiago**
Resolve #3

### CODE REVIEW

**Comment by luchiago** [comment]
@renatamarques97 Please be attentive to spelling during the review and let me know if you need more instructions.

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Installation Guide
```
Comment (renatamarques97) [inline INSTALLATION.md]: Don't you think that it would be better if this guide were in the README.md?

Comment (luchiago) [reply INSTALLATION.md]: I don't think so. I think that README.md should be informed about the project, collaborators and other stuff. Of course, will be a section in README.md redirecting to the installation guide. But if you want so, I can put into README.md

Comment (renatamarques97) [reply INSTALLATION.md]: okay, we can just put a section, sounds great, could you do that? btw thanks a lot for the PR

Comment (luchiago) [reply INSTALLATION.md]: Yes, I can. I'll send right now the section.

**Pull request merged**